ext/hal/ti/simplelink/source/ti/drivers/net/wifi/porting/user.h

```diff
@@ -664,7 +666,19 @@ typedef signed int _SlFd_t;
     \warning
 */
 #ifndef SL_INC_INTERNAL_ERRNO
-#define slcb_SetErrno
+/*
+ * Zephyr Port: use Zephyr SDK's errno.h definitions, and supply those missing
+ * to allow the SimpleLink driver.c to compile
+ * Also, supply the external errno setter function.
+ */
+#include <errno.h>
```

Comment:
However, it would make sense to place all wifi/porting/user.h changes for the port in a separate *commit*, to take it out of the import commit.